src/components/dialog/DialogBase.js

```diff
@@ -20,31 +22,58 @@ export const DialogBase = ({
   onEscKeyDown,
   onOverlayClick,
   scrollable,
+  fadeOutOnClose,
   size,
   initialFocusRef,
 }) => {
   const { ref, FocusRing } = useFocusTrap({ active, initialFocusRef });
 
-  if (!active) {
+  const [delayedActive, setDelayedActive] = useState(false);
```

Comment:
PP: react-transition-group has an event when the dialog has exited, can you use that instead of the `setTimeout`?

http://reactcommunity.org/react-transition-group/transition#Transition-prop-onExited

Reply:
Gonna try it out!

Reply:
Works 👍